lib/services/commandoio.rb

```diff
@@ -58,6 +64,17 @@ def receive_event
     http_post url, params
   end
 
+  # Checks if branch filtering is enabled and if the payload's ref matches
+  #
+  # Returns true or false
+  def filter_branch?
+    commit_branch = (payload['ref'] || '').split('/').last || ''
+    filter_branch = data['filter_branch'].to_s
+
+    # If filtering by branch then don't make a post
+    ((filter_branch.length > 0) && (commit_branch.index(filter_branch) == nil)) ? false : true
```

Comment:
I somewhat prefer the muliple/wildcard matching as is done in https://github.com/github/github-services/pull/1058/files#diff-5dc7176f8aa22691f0ae7be3dc7c2785R32. But, I don't have a strong opinion. :+1: so long as @kdaigle doesn't have a preference either.